README.md

```diff
@@ -18,15 +17,21 @@ The automated scripts run commands as `sudo` and update system libraries and com
 
 In a new container, you'll probably need the following:
 
+## Ubuntu 18+
+```bash
+apt-get update -y && apt-get install -y sudo git python3
+update-alternatives --install /usr/bin/python python /usr/bin/python3 1
```

Comment:
these `update-alternatives` lines exist because none of the new OS provide an alias for python3 and LLVM uses `python` to invoke the interpreter.

Reply:
consider replacing all usage of update-alternatives with a python 3 virtual environment.

Reply:
i think we should let this stay for now and capture the venv change in an issue.

thoughts?

Reply:
For boost, I think passing pass `--with-python=python3` to `bootsrap.sh` will eliminate the need for `/usr/bin/python`.
I believe for clang/llvm, passing `-DPYTHON_EXECUTABLE=/usr/bin/python3` to CMake will do the same.

Reply:
i believe the boost external has been updated. i'll look into adding the llvm one.

Reply:
Looks like the python issue may be resolved by using a later version of cmake. The CMake module that forces externals to need update-alternatives has been deprecated. See https://github.com/Kitware/CMake/blob/master/Modules/FindPythonInterp.cmake#L8.

LLVM 12 (not the `-init` tag) has updated how they find python. See https://github.com/llvm/llvm-project/blob/fed41342a82f5a3a9201819a82bf7a48313e296b/clang/CMakeLists.txt#L134.

@SwooshyCueb I noticed that externals/cmake was just bumped to 3.21.4. Is there a reason why we didn't bump cmake to 3.22.1? If not, I'm going to go on and bump it again in this PR.

Reply:
Then again, i just realized you have bumped the version in all the other projects. I'll leave the version as 3.21.4.

Reply:
I picked the latest 3.21 as the dust is still settling on 3.22. 3.22.1 was released nine days ago.

Reply:
yes, 3.21.4 is plenty new enough, and good for now.

Reply:
Got it.